### ISSUE TITLE
My Game Crashes With: "Cannot invoke "net.minecraft.class_1011.method_4307()" because "this.field_5200" is null"

### ISSUE THREAD

**Issue by IMB11** [IMB11/SkinShuffle#86]
If you recently got a crash that started upon the lines of "Cannot invoke "net.minecraft.class_1011.method_4307()" because "this.field_5200" is null", it's non-fixable. This is an issue with Minecraft. 

I have been trying for the past few months to figure out why this is occuring, but after many iterations, I have not solved the issue, but I have limited the chances it occurs. This has something to do with Minecraft allocating textures, deobfuscated, a typical crash log goes along the lines of:

```log
java.lang.NullPointerException: Cannot invoke "net.minecraft.client.texture.NativeImage.getWidth()" because "this.width" is null
	at net.minecraft.client.texture.NativeImageBackedTexture.method_22793(NativeImageBackedTexture.java:24)
	at com.mojang.blaze3d.systems.RenderSystem.replayQueue(RenderSystem.java:226)
	at com.mojang.blaze3d.systems.RenderSystem.flipFrame(RenderSystem.java:213)
	at net.minecraft.client.util.Window.swapBuffers(Window.java:288)
	at net.minecraft.client.MinecraftClient.render(MinecraftClient.java:1241)
	at net.minecraft.client.MinecraftClient.run(MinecraftClient.java:802)
	at net.minecraft.client.main.Main.main(Main.java:250)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
```

This has a higher chance of occuring on Intel CPUs it seems, especially older generation ones from pre-2015. This may just be a coincidence though, as the crash has rarely occured also with AMD CPUs.

It should be noted that it seems **this crash ONLY OCCURS ON WINDOWS**. If you are on MacOS or Linux, these crashes will not occur.

If you're a developer that has any ideas on how to fix this issue, please let me know.

If you're a player, you're in tough luck. Just hope that it doesn't happen often.

**Comment by IMB11** [comment]
This is fixed in 2.0.4

**Comment by ZKL5859** [comment]
not for me
